api-spec/wallet-api.yaml

```diff
@@ -283,6 +279,70 @@ paths:
           description: Internal Server Error
         '504':
           description: Timeout serving request
+  /wallets/{walletId}/sessions/{orderId}/notifications:
+    post:
```

Comment:
Below in code I see that code take Authorization header to extract wallet token, add security section here to explicitate Authorization header with it's semantic

Reply:
Fixed with ee3d8482f032d1fbfe33d8c6252600bdcfc9fae5